rslib/src/import_export/package/apkg/import/decks.rs

```diff
@@ -46,7 +46,7 @@ impl DeckContext<'_> {
 
     fn import_decks(&mut self, mut decks: Vec<Deck>) -> Result<()> {
         // ensure parents are seen before children
-        decks.sort_unstable_by(|d1, d2| d1.name.as_native_str().cmp(d2.name.as_native_str()));
+        decks.sort_unstable_by_key(|deck| deck.level());
```

Comment:
This was a clever way of avoiding case issues, and performs well to boot - well done!